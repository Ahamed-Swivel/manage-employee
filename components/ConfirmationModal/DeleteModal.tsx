import React from 'react'
import ConfirmationModal from './Modal'

interface DeleteModalProps {
  isModalVisible: boolean;
  onConfirm: () => void;
  onCancel: () => void;
}

const DeleteModal = ({onConfirm, onCancel, isModalVisible}: DeleteModalProps) =>
  <ConfirmationModal
    isModalVisible={isModalVisible}
    title="Confirm Delete"
    message="Are you sure you want to delete this employee?"
    confirmBtnText="Delete"
    variant="danger"
    onConfirm={onConfirm}
    onCancel={onCancel}
  />

export default DeleteModal
