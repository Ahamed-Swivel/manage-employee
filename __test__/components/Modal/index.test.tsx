import { render } from '@testing-library/react'
import '@testing-library/jest-dom'


import DeleteModal from '@/components/ConfirmationModal/DeleteModal'

describe('Modal', () => {
  it('renders Delete modal', () => {
    const component = render(
      <DeleteModal
        isModalVisible={true}
        onConfirm= {() => {return}}
        onCancel= {() => {return}}
      />
    )

    expect(component).toMatchSnapshot()
  })
})
