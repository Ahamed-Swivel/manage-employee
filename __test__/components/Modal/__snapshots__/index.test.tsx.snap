// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components renders EmployeeCard 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="col-md-3"
      >
        <div
          class="mt-3 card"
        >
          <img
            class="card-img-top"
            src="https://randomuser.me/api/portraits/men/92.jpg"
          />
          <div
            class="card-body"
          >
            <div
              class="card-title h5"
            >
              Henri Rodriguez
            </div>
          </div>
          <div
            class="list-group-flush list-group"
          >
            <div
              class="list-group-item"
            >
              <p
                class="card-text"
              >
                Darrin_Rippin@gmail.com
              </p>
            </div>
            <div
              class="list-group-item"
            >
              <p
                class="card-text"
              >
                +94771277218
              </p>
            </div>
            <div
              class="list-group-item"
            >
              <p
                class="card-text"
              >
                Male
              </p>
            </div>
            <div
              class="list-group-item"
            >
              <a
                href="/employee/update/644127d23e6e56a2b8e8b91d"
              >
                <button
                  class="me-2 btn btn-outline-primary"
                  type="button"
                >
                  Update
                </button>
              </a>
              <button
                class="btn btn-outline-danger"
                type="button"
              >
                Delete
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="col-md-3"
    >
      <div
        class="mt-3 card"
      >
        <img
          class="card-img-top"
          src="https://randomuser.me/api/portraits/men/92.jpg"
        />
        <div
          class="card-body"
        >
          <div
            class="card-title h5"
          >
            Henri Rodriguez
          </div>
        </div>
        <div
          class="list-group-flush list-group"
        >
          <div
            class="list-group-item"
          >
            <p
              class="card-text"
            >
              Darrin_Rippin@gmail.com
            </p>
          </div>
          <div
            class="list-group-item"
          >
            <p
              class="card-text"
            >
              +94771277218
            </p>
          </div>
          <div
            class="list-group-item"
          >
            <p
              class="card-text"
            >
              Male
            </p>
          </div>
          <div
            class="list-group-item"
          >
            <a
              href="/employee/update/644127d23e6e56a2b8e8b91d"
            >
              <button
                class="me-2 btn btn-outline-primary"
                type="button"
              >
                Update
              </button>
            </a>
            <button
              class="btn btn-outline-danger"
              type="button"
            >
              Delete
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Components renders EmployeeList 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <table
        class="table table-striped table-bordered table-hover"
      >
        <thead>
          <tr>
            <th>
              Name
            </th>
            <th>
              Email
            </th>
            <th>
              Contact
            </th>
            <th>
              Gender
            </th>
            <th>
              Actions
            </th>
          </tr>
        </thead>
        <tbody>
          <tr>
            <td>
              Henri
            </td>
            <td>
              Darrin_Rippin@gmail.com
            </td>
            <td>
              +94771277218
            </td>
            <td>
              Male
            </td>
            <td>
              <a
                href="/employee/update/644127d23e6e56a2b8e8b91d"
              >
                <button
                  class="me-2 btn btn-outline-primary"
                  type="button"
                >
                  Update
                </button>
              </a>
              <button
                class="btn btn-outline-danger"
                type="button"
              >
                Delete
              </button>
            </td>
          </tr>
        </tbody>
      </table>
    </div>
  </body>,
  "container": <div>
    <table
      class="table table-striped table-bordered table-hover"
    >
      <thead>
        <tr>
          <th>
            Name
          </th>
          <th>
            Email
          </th>
          <th>
            Contact
          </th>
          <th>
            Gender
          </th>
          <th>
            Actions
          </th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td>
            Henri
          </td>
          <td>
            Darrin_Rippin@gmail.com
          </td>
          <td>
            +94771277218
          </td>
          <td>
            Male
          </td>
          <td>
            <a
              href="/employee/update/644127d23e6e56a2b8e8b91d"
            >
              <button
                class="me-2 btn btn-outline-primary"
                type="button"
              >
                Update
              </button>
            </a>
            <button
              class="btn btn-outline-danger"
              type="button"
            >
              Delete
            </button>
          </td>
        </tr>
      </tbody>
    </table>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Components renders Manage Employee - Add new Employee 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="mt-5 mb-5 container"
      >
        <div
          class="row"
        >
          <div
            class="col"
          >
            <h3>
              Add new employee
            </h3>
          </div>
        </div>
        <form
          class="mt-5"
          novalidate=""
        >
          <div
            class="container"
          >
            <div
              class="mb-3 row"
            >
              <div
                class="col-md-4 offset-md-4"
              >
                <div
                  class="mb-3"
                >
                  <label
                    class="form-label"
                    for="formBasicFirstName"
                  >
                    First Name
                  </label>
                  <input
                    class="form-control"
                    id="formBasicFirstName"
                    placeholder="Enter First Name"
                    type="text"
                    value=""
                  />
                </div>
                <div
                  class="mb-3"
                >
                  <label
                    class="form-label"
                    for="formBasicLastName"
                  >
                    Last Name
                  </label>
                  <input
                    class="form-control"
                    id="formBasicLastName"
                    placeholder="Enter Last Name"
                    type="text"
                    value=""
                  />
                </div>
                <div
                  class="mb-3"
                >
                  <label
                    class="form-label"
                    for="formBasicEmail"
                  >
                    Email
                  </label>
                  <input
                    class="form-control"
                    id="formBasicEmail"
                    placeholder="Enter email"
                    type="email"
                    value=""
                  />
                </div>
                <div
                  class="mb-3"
                >
                  <label
                    class="form-label"
                    for="formBasicNumber"
                  >
                    Contact Number
                  </label>
                  <input
                    class="form-control"
                    id="formBasicNumber"
                    placeholder="Enter Contact Number"
                    type="text"
                    value=""
                  />
                </div>
                <div
                  class="mb-3"
                >
                  <label
                    class="form-label"
                    for="formBasicGender"
                  >
                    Gender
                  </label>
                  <select
                    class="form-select"
                    id="formBasicGender"
                  >
                    <option
                      value="M"
                    >
                      Male
                    </option>
                    <option
                      value="F"
                    >
                      Female
                    </option>
                  </select>
                </div>
                <div
                  class="mb-3"
                >
                  <label
                    class="form-label"
                    for="formBasicPhoto"
                  >
                    Photo URL
                  </label>
                  <input
                    class="form-control"
                    id="formBasicPhoto"
                    placeholder="Enter Photo URL"
                    type="url"
                    value=""
                  />
                </div>
                <button
                  class="btn btn-primary"
                  type="submit"
                >
                  Submit
                </button>
              </div>
            </div>
          </div>
        </form>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="mt-5 mb-5 container"
    >
      <div
        class="row"
      >
        <div
          class="col"
        >
          <h3>
            Add new employee
          </h3>
        </div>
      </div>
      <form
        class="mt-5"
        novalidate=""
      >
        <div
          class="container"
        >
          <div
            class="mb-3 row"
          >
            <div
              class="col-md-4 offset-md-4"
            >
              <div
                class="mb-3"
              >
                <label
                  class="form-label"
                  for="formBasicFirstName"
                >
                  First Name
                </label>
                <input
                  class="form-control"
                  id="formBasicFirstName"
                  placeholder="Enter First Name"
                  type="text"
                  value=""
                />
              </div>
              <div
                class="mb-3"
              >
                <label
                  class="form-label"
                  for="formBasicLastName"
                >
                  Last Name
                </label>
                <input
                  class="form-control"
                  id="formBasicLastName"
                  placeholder="Enter Last Name"
                  type="text"
                  value=""
                />
              </div>
              <div
                class="mb-3"
              >
                <label
                  class="form-label"
                  for="formBasicEmail"
                >
                  Email
                </label>
                <input
                  class="form-control"
                  id="formBasicEmail"
                  placeholder="Enter email"
                  type="email"
                  value=""
                />
              </div>
              <div
                class="mb-3"
              >
                <label
                  class="form-label"
                  for="formBasicNumber"
                >
                  Contact Number
                </label>
                <input
                  class="form-control"
                  id="formBasicNumber"
                  placeholder="Enter Contact Number"
                  type="text"
                  value=""
                />
              </div>
              <div
                class="mb-3"
              >
                <label
                  class="form-label"
                  for="formBasicGender"
                >
                  Gender
                </label>
                <select
                  class="form-select"
                  id="formBasicGender"
                >
                  <option
                    value="M"
                  >
                    Male
                  </option>
                  <option
                    value="F"
                  >
                    Female
                  </option>
                </select>
              </div>
              <div
                class="mb-3"
              >
                <label
                  class="form-label"
                  for="formBasicPhoto"
                >
                  Photo URL
                </label>
                <input
                  class="form-control"
                  id="formBasicPhoto"
                  placeholder="Enter Photo URL"
                  type="url"
                  value=""
                />
              </div>
              <button
                class="btn btn-primary"
                type="submit"
              >
                Submit
              </button>
            </div>
          </div>
        </div>
      </form>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Components renders Manage Employee - Update Employee 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="mt-5 mb-5 container"
      >
        <div
          class="row"
        >
          <div
            class="col"
          >
            <h3>
              Update employee: Henri
            </h3>
          </div>
        </div>
        <form
          class="mt-5"
          novalidate=""
        >
          <div
            class="container"
          >
            <div
              class="mb-3 row"
            >
              <div
                class="col-md-4 offset-md-4"
              >
                <div
                  class="mb-3"
                >
                  <label
                    class="form-label"
                    for="formBasicFirstName"
                  >
                    First Name
                  </label>
                  <input
                    class="form-control"
                    id="formBasicFirstName"
                    placeholder="Enter First Name"
                    type="text"
                    value="Henri"
                  />
                </div>
                <div
                  class="mb-3"
                >
                  <label
                    class="form-label"
                    for="formBasicLastName"
                  >
                    Last Name
                  </label>
                  <input
                    class="form-control"
                    id="formBasicLastName"
                    placeholder="Enter Last Name"
                    type="text"
                    value="Rodriguez"
                  />
                </div>
                <div
                  class="mb-3"
                >
                  <label
                    class="form-label"
                    for="formBasicEmail"
                  >
                    Email
                  </label>
                  <input
                    class="form-control"
                    id="formBasicEmail"
                    placeholder="Enter email"
                    type="email"
                    value="Darrin_Rippin@gmail.com"
                  />
                </div>
                <div
                  class="mb-3"
                >
                  <label
                    class="form-label"
                    for="formBasicNumber"
                  >
                    Contact Number
                  </label>
                  <input
                    class="form-control"
                    id="formBasicNumber"
                    placeholder="Enter Contact Number"
                    type="text"
                    value="+94771277218"
                  />
                </div>
                <div
                  class="mb-3"
                >
                  <label
                    class="form-label"
                    for="formBasicGender"
                  >
                    Gender
                  </label>
                  <select
                    class="form-select"
                    id="formBasicGender"
                  >
                    <option
                      value="M"
                    >
                      Male
                    </option>
                    <option
                      value="F"
                    >
                      Female
                    </option>
                  </select>
                </div>
                <div
                  class="mb-3"
                >
                  <label
                    class="form-label"
                    for="formBasicPhoto"
                  >
                    Photo URL
                  </label>
                  <input
                    class="form-control"
                    id="formBasicPhoto"
                    placeholder="Enter Photo URL"
                    type="url"
                    value="https://randomuser.me/api/portraits/men/92.jpg"
                  />
                </div>
                <button
                  class="btn btn-primary"
                  type="submit"
                >
                  Update
                </button>
              </div>
            </div>
          </div>
        </form>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="mt-5 mb-5 container"
    >
      <div
        class="row"
      >
        <div
          class="col"
        >
          <h3>
            Update employee: Henri
          </h3>
        </div>
      </div>
      <form
        class="mt-5"
        novalidate=""
      >
        <div
          class="container"
        >
          <div
            class="mb-3 row"
          >
            <div
              class="col-md-4 offset-md-4"
            >
              <div
                class="mb-3"
              >
                <label
                  class="form-label"
                  for="formBasicFirstName"
                >
                  First Name
                </label>
                <input
                  class="form-control"
                  id="formBasicFirstName"
                  placeholder="Enter First Name"
                  type="text"
                  value="Henri"
                />
              </div>
              <div
                class="mb-3"
              >
                <label
                  class="form-label"
                  for="formBasicLastName"
                >
                  Last Name
                </label>
                <input
                  class="form-control"
                  id="formBasicLastName"
                  placeholder="Enter Last Name"
                  type="text"
                  value="Rodriguez"
                />
              </div>
              <div
                class="mb-3"
              >
                <label
                  class="form-label"
                  for="formBasicEmail"
                >
                  Email
                </label>
                <input
                  class="form-control"
                  id="formBasicEmail"
                  placeholder="Enter email"
                  type="email"
                  value="Darrin_Rippin@gmail.com"
                />
              </div>
              <div
                class="mb-3"
              >
                <label
                  class="form-label"
                  for="formBasicNumber"
                >
                  Contact Number
                </label>
                <input
                  class="form-control"
                  id="formBasicNumber"
                  placeholder="Enter Contact Number"
                  type="text"
                  value="+94771277218"
                />
              </div>
              <div
                class="mb-3"
              >
                <label
                  class="form-label"
                  for="formBasicGender"
                >
                  Gender
                </label>
                <select
                  class="form-select"
                  id="formBasicGender"
                >
                  <option
                    value="M"
                  >
                    Male
                  </option>
                  <option
                    value="F"
                  >
                    Female
                  </option>
                </select>
              </div>
              <div
                class="mb-3"
              >
                <label
                  class="form-label"
                  for="formBasicPhoto"
                >
                  Photo URL
                </label>
                <input
                  class="form-control"
                  id="formBasicPhoto"
                  placeholder="Enter Photo URL"
                  type="url"
                  value="https://randomuser.me/api/portraits/men/92.jpg"
                />
              </div>
              <button
                class="btn btn-primary"
                type="submit"
              >
                Update
              </button>
            </div>
          </div>
        </div>
      </form>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Modal renders Delete modal 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body
    class="modal-open"
    data-rr-ui-modal-open=""
    style="padding-right: 1024px;"
  >
    <div />
    <div
      class="fade modal-backdrop show"
    />
    <div
      aria-modal="true"
      class="fade modal show"
      role="dialog"
      style="display: block; padding-right: 0px;"
      tabindex="-1"
    >
      <div
        class="modal-dialog"
      >
        <div
          class="modal-content"
        >
          <div
            class="modal-header"
          >
            <div
              class="modal-title h4"
            >
              Confirm Delete
            </div>
            <button
              aria-label="Close"
              class="btn-close"
              type="button"
            />
          </div>
          <div
            class="modal-body"
          >
            Are you sure you want to delete this employee?
          </div>
          <div
            class="modal-footer"
          >
            <button
              class="btn btn-secondary"
              type="button"
            >
              Cancel
            </button>
            <button
              class="btn btn-danger"
              type="button"
            >
              Delete
            </button>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div />,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
